docs/packages/TypeScript_ESLint.mdx

```diff
@@ -0,0 +1,130 @@
+---
+id: typescript-eslint
+sidebar_label: typescript-eslint
+---
+
+# `typescript-eslint`
+
+> Tooling which enables you to use TypeScript with ESLint
+
+This package is the main entrypoint that you can use to consume our tooling with ESLint.
+
+This package exports the following:
+
+| name      | description                                                |
+| --------- | ---------------------------------------------------------- |
+| `config`  | A utility function for creating type-safe flat configs     |
+| `configs` | [Our eslint (flat) configs](../linting/Configurations.mdx) |
+| `parser`  | [Our parser](./Parser.mdx)                                 |
+| `plugin`  | [Our plugin](./ESLint_Plugin.mdx)                          |
+
+## Usage
+
+The `tseslint.config` function is a simple utility which does a few things. First and foremost it is a strictly typed function - meaning it allows you to write type-safe configs!
+
+The simplest usage would be:
+
+```js title="eslint.config.js"
+// @ts-check
+
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
+);
+```
+
+You can also declare our plugin and parser in your config via this package. For example this config would enable our plugin, our parser, and type-aware linting with a few of our popular type-aware rules:
+
+```js title="eslint.config.js"
+// @ts-check
+
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  {
+    plugins: {
+      '@typescript-eslint': tseslint.plugin,
+    },
+    languageOptions: {
+      parser: tseslint.parser,,
```

Comment:
```suggestion
      parser: tseslint.parser,
```

Extra comma :slightly_smiling_face:

Reply:
weird that prettier didn't format this out-of-the-box!